backtesting/v2_allocations/allocations/incentives_2024-07-18_2024-08-01.csv

```diff
@@ -1,36 +1,37 @@
 pool_id,chain,symbol,earned_fees,fees_to_vebal,fees_to_dao,total_incentives,aura_incentives,bal_incentives,redirected_incentives,reroute_incentives,last_join_exit
```

Comment:
Is this with the most recent corrected code.  Would be interested in comparing these singe round results.